inc/lift/RequestPool.h

```diff
@@ -45,17 +45,18 @@ class RequestPool {
         const std::string& url) -> RequestHandle;
 
     /**
-     * Produces a new Request with the specified timeout.
+     * Produces a new Request with the specified connection_timeout.
```

Comment:
Should this be `curl_timeout` instead of `connection_timeout`?

Reply:
Yes, indeed!